Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
I'm actually happy to change it back if you'd like. Just thought I'd give @DomT4's cool new :perl thingy and spin, and see if we could "get away with" the default option wording it provides.